src/FObject.cpp

```diff
@@ -161,15 +161,15 @@ void FObject::tick(float delta)
 {
 	if (m_drags)
 	{
-		m_velocity = m_velocity * 0.97f;
+		m_velocity = m_velocity * 0.9f;
 	}
 	
-	if (m_velocity.getLength() < 0.001f)
+	if (m_velocity.getLength() < 0.01f)
 		m_velocity = FVector();
 		
 	if (isGravitational())
 	{
-		FVector down(0.f, 1.5f * delta);
+		FVector down(0.f, 8.9f * delta);
 		accelerate(down);
 	}
 	
```

Comment:
Obviously, these values are much better than the previous ones. I wonder if we should predefine all constant values like mc_<name> and define them in a constructor with a private method. Not sure if it would be worth it, especially writing all variable names and so on, but if we happen to "calibrate" the game in future, we might need it. I am not talking about all classes, not only this one. What do you think? @naslundx